client/src/components/SectionTable/SectionTable.js

```diff
@@ -79,6 +80,28 @@ class SectionTable extends PureComponent {
                         <Fragment />
```

Comment:
I know you didn't add this, but I just noticed it. Seems like maybe it shouldn't be here... 
@devsdevsdevs do you know why we add a Fragment if the prereq link doesn't exist?